simtools/db_handler.py

```diff
@@ -20,653 +22,879 @@
 from simtools.util import names
 from simtools.util.model import getTelescopeClass
 
-__all__ = ['getArrayDB']
+__all__ = [
+    'getArrayDB',
+    'getModelParameters',
+    'readMongoDB',
+    'writeModelFile',
+    'getModelFile',
+    'getFileMongoDB',
+    'writeFileFromMongoToDisk',
+    'copyTelescope',
+    'deleteQuery',
+    'updateParameter',
+    'addParameter',
+    'addNewParameter'
+]
```

Comment:
I think we should keep only DatabaseHandler in this list. All these functions are class methods, so they should not be here.

Reply:
Yea, that's a remnant from before I turned it into a class. Will fix now and merge.